worlds/witness/options.py

```diff
@@ -210,50 +248,60 @@ class TrapWeights(OptionDict):
 
 
 class PuzzleSkipAmount(Range):
-    """Adds this number of Puzzle Skips into the pool, if there is room. Puzzle Skips let you skip one panel.
-    Works on most panels in the game - The only big exception is The Challenge."""
+    """
+    Adds this number of Puzzle Skips into the pool, if there is room. Puzzle Skips let you skip one panel.
+    """
     display_name = "Puzzle Skips"
     range_start = 0
     range_end = 30
     default = 10
 
 
 class HintAmount(Range):
-    """Adds hints to Audio Logs. If set to a low amount, up to 2 additional duplicates of each hint will be added.
-    Remaining Audio Logs will have junk hints."""
+    """
+    Adds hints to Audio Logs. If set to a low amount, up to 2 additional duplicates of each hint will be added.
+    Remaining Audio Logs will have junk hints.
+    """
     display_name = "Hints on Audio Logs"
     range_start = 0
     range_end = 49
     default = 12
 
 
 class AreaHintPercentage(Range):
-    """There are two types of hints for The Witness.
+    """
+    There are two types of hints for The Witness.
     "Location hints" hint one location in your world / containing an item for your world.
-    "Area hints" will tell you some general info about the items you can find in one of the
-    main geographic areas on the island.
-    Use this option to specify how many of your hints you want to be area hints. The rest will be location hints."""
+    "Area hints" will tell you some general info about the items you can find in one of the main geographic areas on the island.
+    Use this option to specify how many of your hints you want to be area hints. The rest will be location hints.
+    """
     display_name = "Area Hint Percentage"
     range_start = 0
     range_end = 100
     default = 33
 
 
 class LaserHints(Toggle):
-    """If on, lasers will tell you where their items are if you walk close to them in-game.
-    Only applies if laser shuffle is enabled."""
+    """
+    If on, lasers will tell you where their items are if you walk close to them in-game.
+    Only applies if laser shuffle is enabled.
+    """
     display_name = "Laser Hints"
 
 
 class DeathLink(Toggle):
-    """If on: Whenever you fail a puzzle (with some exceptions), everyone who is also on Death Link dies.
-    The effect of a "death" in The Witness is a Bonk Trap."""
+    """
+    If on: Whenever you fail a puzzle (with some exceptions), everyone who is also on Death Link dies.
+    The effect of a "death" in The Witness is a Bonk Trap.
+    """
     display_name = "Death Link"
 
 
 class DeathLinkAmnesty(Range):
-    """Number of panel fails to allow before sending a death through Death Link.
-    0 means every panel fail will send a death, 1 means every other panel fail will send a death, etc."""
+    """
+    Number of panel fails to allow before sending a death through Death Link.

```

Comment:
Could also be "How many panel fails"
```suggestion
    The number of panel fails to allow before sending a death through Death Link.
```